library/photon

```diff
@@ -11,3 +14,14 @@ GitCommit: 3c1e91142077d224ce57f8890e3ae12f64a92981
 Tags: 1.0, 1.0-20180816
 GitFetch: refs/heads/1.0-20180816
 GitCommit: fda590b062d1ac4d575c67fff4847b483f5097cd
+
+Tags: dev-20180901
+SharedTags: dev
+GitFetch: refs/heads/x86_64/dev-20180901
+GitCommit: 9dda8da4767f493b80ac75a5f52d783c49f27718
+
+Tags: dev-20180904
```

Comment:
This seems really strange (and is likely to be really confusing for users); is there any particular reason one of these can't be regenerated so they've got matching datestamps?

What this is going to look like as-is is that the page at https://hub.docker.com/_/photon/ will list both tags under the list of supported tags, with no obvious indication that each supports an entirely separate architecture (so users on `amd64` are likely to try running `dev-20180904` and it'll just fail with a cryptic error message about unsupported platform).